package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cdp-client",
-  "version": "2.0.3",
+  "version": "2.0.4",
```

Comment:
I'd increase the minor version as well. Multi-app and event support is a bigger thing than just a patch